src/RoomHour.java

```diff
@@ -0,0 +1,17 @@
+public class RoomHour {
```

Comment:
If you decided to use the RoomHour class as sth separate - maybe it would make sense to use it as an inner class -> as it is strictly related to the Rooms enum